frontend/src/modules/Dashboard/Styles/GroupCard.style.tsx

```diff
@@ -1,11 +1,24 @@
 import styled from 'styled-components'
 import { FiClock } from 'react-icons/fi'
+import { ReactComponent as GroupsIcon } from '@assets/img/groupsicon.svg'
+import { ReactComponent as PlusIcon } from '@assets/img/plusicon.svg'
+import { ReactComponent as NewlyMatchable } from '@assets/img/newlymatchable.svg'
 import { colors, h3, h4, StyledComponent } from '@core'
+import { Props } from 'react'
 
-export const StyledContainer = styled.div`
+export const StyledGroupsIcon = styled(GroupsIcon)`
+  padding-right: 1rem;
+`
+
+export const StyledPlusIcon = styled(PlusIcon)`
+  padding-right: 1rem;
+`
+export const StyledNewlyMatchable = styled(NewlyMatchable)``
+
+export const StyledContainer = styled.div<Pick<ColorProps, 'color'>>`
   height: 300px;
   width: 320px;
-  background: ${colors.paleviolet};
+  background: ${(p) => p.color};
```

Comment:
Interesting, I haven't actually used Styled Components this way yet

Reply:
This is giving d3 data joins lol